[2023-11-25 08:41:12,881] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:4001
 * Running on http://192.168.68.102:4001 (Press CTRL+C to quit) - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:41:15,010] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:41:15] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:41:15,294] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:41:15] "GET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:41:15,296] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:41:15] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:41:15,300] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:41:15] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:41:16,713] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:41:16] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:41:16,949] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:41:16] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:48:41,885] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:4001
 * Running on http://192.168.68.102:4001 (Press CTRL+C to quit) - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:48:46,852] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:48:46] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:48:47,302] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:48:47] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:48:47,313] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:48:47] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:48:47,317] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:48:47] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:48:47,971] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:48:47] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:48:48,144] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:48:48] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:48:55,808] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:48:55] "POST /raa HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:49:15,388] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:4001
 * Running on http://192.168.68.102:4001 (Press CTRL+C to quit) - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:49:19,238] ERROR log_exception() L1449 Exception on /raa [POST] - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py L1449
Traceback (most recent call last):
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_openapi3/scaffold.py", line 191, in wrapper
    resp = _do_wrapper(
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_openapi3/do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
  File "/Users/tairik/PycharmProjects/mvp-tairik/rest_api_pets/app.py", line 67, in get_raa
    prediction = loaded_model.predict(X_entrada)  # Passing in variables for prediction
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/sklearn/svm/_base.py", line 818, in predict
    y = super().predict(X)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/sklearn/svm/_base.py", line 431, in predict
    X = self._validate_for_predict(X)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/sklearn/svm/_base.py", line 611, in _validate_for_predict
    X = self._validate_data(
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/sklearn/base.py", line 605, in _validate_data
    out = check_array(X, input_name="X", **check_params)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/sklearn/utils/validation.py", line 957, in check_array
    _assert_all_finite(
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/sklearn/utils/validation.py", line 122, in _assert_all_finite
    _assert_all_finite_element_wise(
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/sklearn/utils/validation.py", line 171, in _assert_all_finite_element_wise
    raise ValueError(msg_err)
ValueError: Input X contains NaN.
SVC does not accept missing values encoded as NaN natively. For supervised learning, you might want to consider sklearn.ensemble.HistGradientBoostingClassifier and Regressor which accept missing values encoded as NaNs natively. Alternatively, it is possible to preprocess the data, for instance by using an imputer transformer in a pipeline or drop samples with missing values. See https://scikit-learn.org/stable/modules/impute.html You can find a list of all estimators that handle NaN values at the following page: https://scikit-learn.org/stable/modules/impute.html#estimators-that-handle-nan-values
[2023-11-25 08:49:19,310] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:49:19] "[35m[1mPOST /raa HTTP/1.1[0m" 500 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:50:01,789] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:4001
 * Running on http://192.168.68.102:4001 (Press CTRL+C to quit) - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:50:08,540] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:50:08] "POST /raa HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:51:15,284] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:4001
 * Running on http://192.168.68.102:4001 (Press CTRL+C to quit) - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:51:17,814] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:51:17] "POST /raa HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-25 08:51:54,095] INFO _log() L224  192.168.68.102 - - [25/Nov/2023 08:51:54] "POST /raa HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
